### ISSUE TITLE
minishift oc-env should always add the minishift ip to no_proxy

### ISSUE THREAD

**Issue by svenstaro** [minishift/minishift#2269]
If we try to use oc behind a corporate firewall without appending the minishift ip the no_proxy, oc will fail to connect as it will use the https_proxy to connect to the local openshift instance which will fail. I think it's generally safe to put the minishift ip into no_proxy for minishift oc-env.

**Comment by LalatenduMohanty** [comment]
@svenstaro Thanks for bringing it up. We have `--no-proxy` in `minishift docker-env`  but not in `oc-env`.

```
$ minishift docker-env --help
Sets Docker environment variables, similar to '$(docker-machine env)'.

Usage:
  minishift docker-env [flags]

Flags:
  -h, --help           help for docker-env
      --no-proxy       Add the virtual machine IP to the NO_PROXY environment variable.
      --shell string   Force setting the environment for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh]. Default is auto-detect.
  -u, --unset          Clear the environment variable values instead of setting them.
```

**Comment by LalatenduMohanty** [comment]
For example 

```
$ minishift docker-env --no-proxy
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://192.168.64.3:2376"
export DOCKER_CERT_PATH="/Users/lmohanty/.minishift/certs"
export NO_PROXY="192.168.64.3"
# Run this command to configure your shell:
# eval $(minishift docker-env --no-proxy)
```

**Comment by svenstaro** [comment]
Closed in #2280
